README.md

```diff
@@ -389,30 +391,72 @@ mkdir gx_logs
 docker run -d -p 8080:80 -p 8021:21 -e "GALAXY_LOGGING=full" -v `pwd`/gx_logs:/home/galaxy/logs bgruening/galaxy-stable
 ```
 
+## Running on an external cluster (DRM)  <a name="Running-on-an-external-cluster-(DRM)" />[[toc]](#toc)
 
-## Using an external Slurm cluster <a name="Using-an-external-Slurm-cluster" /> [[toc]](#toc)
+### Basic setup for the filesystem  <a name="Basic-setup-for-the-filesystem" /> [[toc]](#toc)
+The docker container sets up all its files on the /export directory, but this directory does not exist on the cluster filesystem. This can be solved with symbolic links on the cluster filesystem but it can also be solved within the container itself.
```

Comment:
The following is only valid if you don't have /export/ mounted on your cluster right? Can we highlight this a little bit more maybe making an other sub-heading ...